answer/src/main/java/org/exoplatform/platform/qa/ui/answer/pageobject/QuestionManagement.java

```diff
@@ -369,14 +359,37 @@ public void activeQuestion(Boolean isActive) {
    */
   public void rateQuestion(int number) {
     info("Rate a question");
-    String st = Integer.toString(number);
-    if (number != 0) {
-      WebElement e1 = evt.waitForAndGetElement(By.xpath(ELEMENT_QUESTION_RATE_ITEM.replace("$index", st)),
-                                               testBase.getDefaultTimeout(),
-                                               1,
-                                               2);
-      ((JavascriptExecutor) testBase.getExoWebDriver().getWebDriver()).executeScript("arguments[0].click();", e1);
-      evt.waitForAndGetElement(ELEMENT_QUESTION_RATE_NUMBER.replace("$index", st));
+    switch (number) {
+    case 1:
+      ELEMENT_RATE_QUESTION_1_UNVOTED.click();
+      assertEquals("rgba(255, 196, 13, 1)", ELEMENT_RATE_QUESTION_1_VOTED.getCssValue("color"));
```

Comment:
Can be broken depending on the applied skin, no ?

Reply:
no, i verified from this issue https://jira.exoplatform.org/browse/PFR-1340

Reply:
:+1: